app/assets/stylesheets/partials/_content_page.scss

```diff
@@ -0,0 +1,27 @@
+  .content {
+    max-width: 33em;
+    margin-left: 40em;
```

Comment:
There should be an additional container around .content with margin: auto to center the expected content area. Such that:
```
|container margin auto|margin-left: 40em|text|margin-right:16em|container margin auto|
|---------------------|-----------------|----|-----------------|---------------------|
```
At least according to the mock, the total container width would be about 90em. This is all pretty rigid and we should scale the proportions accordingly until a certain breakpoint.